test/ui/FacetValueSuggestionsTest.ts

```diff
@@ -67,6 +67,9 @@ export function FacetValueSuggestionsTest() {
       omniboxInstance = Mock.mock(Omnibox);
       omniboxInstance.suggestionAddon = Mock.mock(QuerySuggestAddon);
       omniboxInstance.suggestionAddon.getSuggestion = jasmine.createSpy('getSuggestions');
+      omniboxInstance.magicBox = {
+        blur: jasmine.createSpy('blur')
```

Comment:
Should there be a test to check that onBlur, it closes the dropdown of the magicbox? or triggers a 'blur' event?